nomad/assets/dashboards/overview.json

```
@@ -0,0 +1,867 @@
+            "width": 20,
+            "x": 63,
+            "y": 71
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "avg:nomad.client.consul.services{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "# Consul Services",
+            "type": "query_value",
+            "width": 20,
+            "x": 42,
+            "y": 58
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "avg:nomad.client.consul.check_registrations{$host}.as_count()"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "# Checks Registrations",
+            "type": "query_value",
+            "width": 20,
+            "x": 42,
+            "y": 84
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "avg:nomad.client.consul.checks{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "center",
+            "title_size": 16,
+            "title_text": "# Consul Checks",
+            "type": "query_value",
+            "width": 20,
+            "x": 42,
+            "y": 71
+        },
+        {
+            "height": 10,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 2,
+                "requests": [
+                    {
+                        "aggregator": "avg",
+                        "q": "avg:nomad.raft.leader.lastContact.avg{$host}"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Time Since Contact With Leader",
+            "type": "query_value",
+            "width": 20,
+            "x": 63,
+            "y": 84
+        },
+        {
+            "bgcolor": "white",
```

```suggestion
            "bgcolor": "gray",
```

Lets keep section headers as gray. It makes it easier to read the different categories of graphs